Commons/bukkit/src/main/java/tc/oc/commons/bukkit/ticket/TicketBooth.java

```diff
@@ -198,20 +198,21 @@ public void leaveGame(Player player, boolean returnToLobby) {
 
         final PlayerId playerId = userStore.playerId(player);
         final Game game = currentGame(playerId);
-        if(game == null) {
-            audience.sendMessage(gameFormatter.notPlaying());
```

Comment:
Just comment out the not playing message, I'm pretty sure it will still work?

Reply:
Do you want it to only not display the message? When I initially considered this fix I wanted it to retain its functionality and not display the message at the same time